anvil_consortium_manager/views.py

```diff
@@ -389,6 +398,33 @@ def get(self, request, *args, **kwargs):
         # Add a success message.
         messages.add_message(self.request, messages.SUCCESS, self.message_success)
 
+        # Call account adapter after verify hook
+        adapter_class = get_account_adapter()
+        adapter_instance = adapter_class()
+
+        try:
+            adapter_instance.after_account_link_verify(user=account.user)
+        except Exception as e:
+            # Log but do not stop execution
+            logger.exception(f"[AccountLinkVerify] {self.log_message_after_account_link_failed}: {e}")
+
+            # Get the exception type and message
+            error_description = f"{type(e).__name__}: {str(e)}"
+
+            # Send a mail about issue if account veriy notification email is set
+            if get_account_adapter().account_verify_notification_email:
```

Comment:
Can you just use `adapter_instance` instead of `get_account_adapter` here? may be slightly faster